gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcHeaderInterceptor.java

```diff
@@ -65,7 +64,7 @@ public GrpcHeaderInterceptor(Map<String, String> staticHeaders) {
       // Specifically the User-Agent will be provided via ManagedChannelBuilder#userAgent(String)
       // during channel construction (calling #getUserAgentHeader() to get the value, initialized
       // here).
-      if (headerKey.equals(GrpcUtil.USER_AGENT_KEY)) {
+      if ("user-agent".equals(headerKey.name())) {
```

Comment:
Why are we replacing `GrpcUtil.USER_AGENT_KEY` with a hardcoded string?

Reply:
That was a mistake (original version). `GrpcUtil` resides under `io.grpc.internal` (**internal**!) package, which is distributed in main `grpc-core.jar` for technical reasons. In case if we consume grpc as bazel workspace, the internal package is a different target with its own dependencies. In general, consumers of grpc should not depend on anything inside `io.grpc.internal`. Using string literal should safe here, because `"user-agent"` is not a magic constant but a part of http spec.

Reply:
OK, although maybe it is still worth putting it into a USER_AGENT_KEY static field? Not 100% convinced but that is my mild preference.